kedro/framework/cli/starters.py

```diff
@@ -359,20 +367,49 @@ def list_starters():
         )
 
 
-def _get_addons_from_cli_input(
-    selected_addons: str, config: dict[str, str]
-) -> dict[str, str]:
-    """Inserts add-on selection from the CLI input in the project
-    configuration, if it exists. Replaces add-on strings with the
-    corresponding prompt number.
+def _get_config(
+    prompts_required,
+    config_path,
+    cookiecutter_context,
+    selected_addons,
+    project_name,
+):
```

Comment:
I think it would be good to have a doc string here, explaining what the method does.